spec/outputs/elasticsearch_spec.rb

```diff
@@ -56,7 +56,7 @@
         response.read_body { |chunk| data << chunk }
         result = LogStash::Json.load(data)
         count = result["count"]
-        insist { count } == event_count
+        expect { count } == event_count
```

Comment:
we have traditionally been using `insist` for such assertions. I think we should keep it with `insist` for the sake of keeping this PR's main goal to add the `_routing` functionality